### PR TITLE
Fixed mobile footer links and added some padding.

### LINKED ISSUE #11831: Inaccessible links on Ubuntu.com footer (mobile only).
## Summary

[On a mobile screen, at the footer, the drop-downs containing links to other pages or sections on the website can't be clicked, so the links aren't visible.]

[Also, the drop-down texts are too small and they touch the edge of the screen. The color also makes them appear as though they were disappearing. This is bad for accessibility.]

## Process

[1. Fix the drop-down links.]
[2. Increase the font-size and make the texts darker.]
[3. Add padding-inline to shift the text away from the screen edge so they're easily read.]

## Current and expected result

[Working drop-downs.]
[Improved accessibility.]

## Browser details

[Tested on Chrome and Safari]

### PR DESCRIPTION
**Pull request by publikphigor**
## Done

- [Increased the font-size of the drop-downs.]
- [Added padding-inline to the drop-downs.]

## Issue / Card
Fixes #11831 

## Screenshots
image

### CODE REVIEW

File: static/sass/_pattern_footer.scss [hunk 1]
```diff
@@ -97,12 +97,14 @@
     background-size: 13px 13px;
     border-top: 1px solid $color-mid-light;
     color: $color-dark;
-    font-size: 0.8125rem;
+    font-size: 1.2rem;
     margin-top: 0;
     max-width: inherit;
     padding: $sp-medium 0;
+    padding-inline: 1rem;
```
Comment (anthonydillon) [inline static/sass/_pattern_footer.scss:104]: I would suggest you simply change the original padding styling on line 103 to `padding: $sp-medium`

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Thank you for contributing @publikphigor. Were you able to look into the click accordion issue? One small comment in the code.

**Comment by publikphigor** [comment]
I found the accordion.js and all look good. I also checked the console and it doesn't throw any error when I click the links.
I suspect they're not selected to run the functions in accordion.js.

**Comment by anthonydillon** [comment]
Great, thanks @publikphigor. I have created a new issue to [tackle the dropdown action](https://github.com/canonical-web-and-design/ubuntu.com/issues/11834). Just waiting on a design review now.

**Comment by lyubomir-popov** [comment]
@anthonydillon how do I QA this? There's no demo link.

**Comment by sowasred2012** [comment]
@lyubomir-popov here's a demo: https://ubuntu-com-11832.demos.haus/

**Comment by anthonydillon** [comment]
No this should be fixed in this branch. Would you be ok to look at this @publikphigor

**Comment by publikphigor** [comment]
I'll look into it @anthonydillon

**Comment by sowasred2012** [comment]
@publikphigor hope you don't mind, I made a couple of updates to allow the accordions to open on click - I don't think this is ready yet, I was hoping to get more design guidance from @davegoddard42:

- there are two different font weights going on, 100 (for accordion CTAs) vs 300 for every other link
- I'm not sure we've discussed the design of the links contained in the accordions, at the moment they're just a darker color than standard links (`#111` vs `#666`)

Peek 2022-07-22 17-16

**Comment by lyubomir-popov** [comment]
> there are two different font weights going on, 100 (for accordion CTAs) vs 300 for every other link

that’s because the css is overriding the font size of h2s, but not the font weight. Ideally we should have a “p-paragraph that does the opposite of”p-heading—-“, to avoid partial overrides that kead to regressions like this.

**Comment by sowasred2012** [comment]
@lyubomir-popov would you mind giving this another design review?

**Comment by davegoddard42** [comment]
LGTM @sowasred2012 - nice work.

**Review by davegoddard42 — APPROVED** [review]
Outstanding design items addressed - looks good

**Pull request merged**